src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -275,9 +272,9 @@ def delete_arc_agents(release_namespace, kube_config, kube_context, configuratio
             telemetry.set_user_fault()
```

Comment:
we can simply remove all the `telemetry.set_user_fault()` and  `telemetry.set_failure()` lines once raising the new error types, the new error types will call `telemetry.set_user_fault()` or `telemetry.set_failure()` according to the error type itself in CLI core.

Reply:
We don't now. Had missed one.